packages/debugger-extension/src/index.ts

```diff
@@ -478,6 +478,19 @@ const variables: JupyterFrontEndPlugin<void> = {
         });
       }
     });
+
+    commands.addCommand(CommandIDs.copyToClipboard, {
+      label: 'Copy to Clipboard',
+      caption: trans.__('Copy value to clipboard'),
+      isEnabled: () => !!service.session?.isStarted,
+      isVisible: () => handler.activeWidget instanceof NotebookPanel,
+      execute: async () => {
+        const value = service.model.variables.selectedVariable!.value;
+        if (value) {
+          await navigator.clipboard.writeText(value);
```

Comment:
We may prefer to use the `Clipboard` utiility from apputils.

Ideally we would be able to fetch a MIME bundle and copy rich data (if permitted). This way when pasting a dataframe to e.g. Word we would have a choice to paste as a rich table, markdown table or plain text. We had a similar discussion for "Copy Output to Clipboard" (see https://github.com/jupyterlab/jupyterlab/pull/10282#issuecomment-8472397860) but deferred due to browser support limitations (https://github.com/jupyterlab/jupyterlab/pull/10282#issuecomment-853364314). Since it was over a year ago we can re-evaluate if you are interested (it's fine if not - just a suggestion).

Reply:
Thanks, I didn't know about this `Clipboard\` utility.